workers/unity/Packages/com.improbable.gdk.buildsystem/WorkerBuilder.cs

```diff
@@ -71,6 +75,30 @@ public static void Build()
             }
         }
 
+        public static void AssertWorkerCanBuildForEnvironment(string workerType, BuildEnvironment targetEnvironment)
```

Comment:
Super nit: Given the name of this function, I would expect it to actually `assert` rather than throw an exception

Reply:
I confirm I'm using the word "Assert" liberally in this context, happy to accept suggestions

Reply:
`Verify`?